.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
+node_js:
+  - "stable"
+install: npm install
+
+services:
+  - postgresql
+
```

Comment:
Add the codeclimate repo token

Reply:
ok seen thanks

Reply:
@Adekoreday Here you go `205b460fb18d22d3db847cb98cc3fbbee316a0624d59024b5e15728eb78ef38e`

Reply:
seen thanks